src/token/erc721/erc721.cairo

```diff
@@ -410,20 +450,20 @@ mod ERC721Component {
         /// Mints `token_id` and transfers it to `to`.
         /// Internal function without access restriction.
         ///
+        /// WARNING: Usage of this method is discouraged, use `_safe_mint` whenever possible
+        ///
```

Comment:
Since users can mint to contracts unable to handle the token when no using safe_mint, I think it should. It is also consistent with the Solidity version.